Source/cmake/WebKitCompilerFlags.cmake

```diff
@@ -460,6 +460,19 @@ int main() {
     unset(CMAKE_REQUIRED_FLAGS)
 endif ()
 
+if (WTF_OS_LINUX)
```

Comment:
Surely you can run this test on every OS? There's nothing Linux-specific about it.

Reply:
I realised COCOA is not an OS, it's a PLATFORM, so I reworked the patch. To make the patch symmetric and guarded the CMake check with `NOT WTF_PLATFORM_COCOA`.

Reply:
Looks good.